pages/api/load.ts

```diff
@@ -1,23 +1,61 @@
-import { NextApiRequest, NextApiResponse } from 'next';
-import { encodePayload, getBCVerify, setSession } from '../../lib/auth';
+import { NextApiRequest, NextApiResponse } from "next";
+import { checkAppExtensionsExist } from "@lib/appExtensions";
+import { getStoreToken } from "@lib/dbs/firebase";
+import { encodePayload, getBCVerify, setSession } from "../../lib/auth";
 
 const buildRedirectUrl = (url: string, encodedContext: string) => {
-    const [path, query = ''] = url.split('?');
-    const queryParams = new URLSearchParams(`context=${encodedContext}&${query}`);
+  const [path, query = ""] = url.split("?");
```

Comment:
🍹 Also noticing indents getting reformatted to 2 spaces. I think we're using 4 in this repo? (cc: @bc-chaz)